### PR TITLE
Issue #85, User can now only select One day In the nav drawer.

### PR DESCRIPTION
**Pull request by t1miller**
#85

I spent a good 20 minutes trying to figure out why selecting today kept loading Wednesday's deals instead of loading the day "Today's" deals, I then realized what the word Today meant. haha

### CODE REVIEW

**Comment by adamjdorwart** [comment]
That's pretty funny. Hey btw I remember seeing a bug that we should probably fix in this PR. When you have a specific day of the week selected, it's possible to unselect it and then no deals show up because there isn't any filter criteria.

**Comment by AdamDorwart** [comment]
That's pretty funny. Hey btw I remember seeing a bug that we should probably fix in this PR. When you have a specific day of the week selected, it's possible to unselect it and then no deals show up because there isn't any filter criteria.

**Comment by t1miller** [comment]
I got fixed that in this pull request. the only way to unselect a day is to select another day.

**Comment by t1miller** [comment]
*i did fix that in this pull request

File: app/src/main/java/com/dgaf/happyhour/Controller/DrawerFragment.java [hunk 1]
```diff
@@ -328,27 +395,20 @@ public void onCheckedChanged(CompoundButton compoundButton, boolean enabled) {
...
 
-    private void disableTodayToggle() {
-        today.setChecked(false);
-        today.setBackgroundResource(R.drawable.ic_today);
-        queryParameters.getDayOfWeekMask().unselectToday();
     }
 
     @Override
-    public void notifyEmpty() {}
+    public void notifyEmpty() {
```
Comment (AdamDorwart) [inline app/src/main/java/com/dgaf/happyhour/Controller/DrawerFragment.java:385]: Should we still have this?

File: app/src/main/java/com/dgaf/happyhour/Controller/DrawerFragment.java [hunk 2]
```diff
@@ -328,27 +395,20 @@ public void onCheckedChanged(CompoundButton compoundButton, boolean enabled) {
...
+    }
 
     @Override
     public void notifyNotEmpty() {}
```
Comment (AdamDorwart) [inline app/src/main/java/com/dgaf/happyhour/Controller/DrawerFragment.java:389]: And this?

File: app/src/main/java/com/dgaf/happyhour/Controller/DrawerFragment.java (hunk 1 above)
Comment (t1miller) [inline app/src/main/java/com/dgaf/happyhour/Controller/DrawerFragment.java:385]: We have to. Its part of the interface we are implementing to get the toast feedback.

**Pull request merged**